### PR TITLE
Update to latest ACK version to make the pipeline work again

### PR DESCRIPTION
**Pull request by muvaf**
### Description of your changes

After a few fixes in code-generator that make the Crossplane integration work again (thanks @AaronME  and @haarchri  !) , https://github.com/crossplane/provider-aws/pull/992 and a few fixes in the first commit of this PR, we are able to update code-generator to the latest version.

Note to reviewers, only the first commit contains handwritten changes.

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

`make reviewable`

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: apis/cloudfront/v1alpha1/zz_types.go [hunk 1]
```diff
@@ -1112,35 +1204,41 @@ type RealtimeLogConfigs struct {
...
 	DomainName *string `json:"domainName,omitempty"`
 
-	OriginAccessIDentity *string `json:"originAccessIDentity,omitempty"`
+	OriginAccessIdentity *string `json:"originAccessIdentity,omitempty"`
 }
 
+// +kubebuilder:skipversion
 type S3OriginConfig struct {
-	OriginAccessIDentity *string `json:"originAccessIDentity,omitempty"`
+	OriginAccessIdentity *string `json:"originAccessIdentity,omitempty"`
```
Comment (negz) [inline apis/cloudfront/v1alpha1/zz_types.go:1223]: This and the above will be breaking changes. It's an alpha API so we can do that, but it's probably worth a prominent release note at least.

File: apis/ec2/v1alpha1/zz_types.go [hunk 2]
```diff
@@ -1711,8 +1955,9 @@ type ManagedPrefixList struct {
 	Tags []*Tag `json:"tags,omitempty"`
 }
 
+// +kubebuilder:skipversion
 type ModifyVPNTunnelOptionsSpecification struct {
-	DPDTimeoutAction *string `json:"dPDTimeoutAction,omitempty"`
+	DPDTimeoutAction *string `json:"dpdTimeoutAction,omitempty"`
```
Comment (negz) [inline apis/ec2/v1alpha1/zz_types.go:1960]: Another breaking change.

File: package/crds/ec2.aws.crossplane.io_addresses.yaml [hunk 3]
```diff
@@ -18,46 +18,6 @@ spec:
     singular: address
   scope: Cluster
   versions:
-  - name: v1alpha1
```
Comment (negz) [inline package/crds/ec2.aws.crossplane.io_addresses.yaml:21]: Did we expect these versions to disappear?

Comment (haarchri) [reply package/crds/ec2.aws.crossplane.io_addresses.yaml:21]: @muvaf same question - in a few ec2 crds stuff is removed is this expected ? the rest in this PR looks good - tested also a few resources

Comment (muvaf) [reply package/crds/ec2.aws.crossplane.io_addresses.yaml:21]: Yes, we added the `skipversion` tags to get those removed. See https://github.com/crossplane/provider-aws/issues/876 for details.

File: apis/ec2/v1alpha1/zz_types.go (hunk 2 above)
Comment (muvaf) [reply apis/ec2/v1alpha1/zz_types.go:1960]: It seems like we don't generate the CRD that was supposed to use this struct, hence it doesn't show up in any CRDs.

**Comment by muvaf** [comment]
Closing this in favor of @AaronME 's https://github.com/crossplane/provider-aws/pull/920 . I swear I looked for that PR and couldn't find it but then saw it while I was looking for https://github.com/crossplane/provider-aws/issues/876 . The PR is exactly the same I believe, so same review comments apply there as well.

**Pull request closed without merging**